src/test/scala/uk/gov/hmrc/test/ui/pages/AuthLoginPage.scala

```diff
@@ -21,46 +21,67 @@ import uk.gov.hmrc.test.ui.conf.TestConfiguration
+  val frontEndUPEUrl: String =
+    s"$rootUrl" + "business-matching/ultimate-parent/registered-in-uk"
+  val frontEndUPEOrgTypeUrl: String =
+    s"$rootUrl" + "business-matching/ultimate-parent/uk-based/entity-type"
+  val redirectUrlField: String = "redirectionUrl"
+  val redirectionUrlField: String = "#redirectionUrl"
+  val credIdField: String = "authorityId"
+  val frontEndNameUrl: String =
+    s"$rootUrl" + "business-matching/ultimate-parent/no-id/input-name"
+  val frontEndCAUrl: String =
+    s"$rootUrl" + "business-matching/ultimate-parent/no-id/check-answers"
+  val frontEndUPERegTypeUrl: String =
+    s"$rootUrl" + "business-matching/ultimate-parent/registered-in-uk"
+  val frontEndNFMNameUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/input-name"
+  val frontEndNFMOrgTypeUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/uk-based/entity-type"
+  val frontEndNFMAddressUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/input-address "
+  val frontEndNFMContactNameUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/input-business-name"
+  val frontEndNFMContactEmailUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/input-email"
+  val frontEndNFMTelephoneUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/telephone"
+  val frontEndNFMTelephoneInputUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/input-telephone"
+  val frontEndFDGroupStatusUrl: String =
+    s"$rootUrl" + "further-details/group-status"
+  val frontEndFDAccountPeriod: String =
+    s"$rootUrl" + "further-details/accounting-period"
+  val frontEndCDAddressUrl: String =
+    s"$rootUrl" + "contact-details/address/input"
+  val frontEndNFMCAUrl: String =
+    s"$rootUrl" + "business-matching/filing-member/no-id/check-answers"
+  val frontEndFDCAUrl: String = s"$rootUrl" + "further-details/check-answers"
+  val frontEndCDCAUrl: String = s"$rootUrl" + "contact-details/check-answers"
+  val frontEndFinalReviewCAUrl: String =
+    s"$rootUrl" + "review-submit/check-answers"
+  val frontEndSubUrl: String = s"$rootUrl" + "review-submit/confirmation"
+  val frontEndDashboardUrl: String = s"$rootUrl" + "pillar2-top-up-tax-home"
+  val frontEndASAUrl: String = s"$rootUrl" + "asa/input-pillar-2-id"
+  val rfmUrl: String =
+    s"$rootUrl" + "replace-filing-member/security/enter-pillar2-id"
+  val enrolmentKeyField: String = "enrolment[0].name"
+  val identifierNameField: String = "input-0-0-name"
+  val identifierValueField: String = "input-0-0-value"
+  val delegatedEnrolmentKeyField: String = "delegatedEnrolment[0].key"
+  val delegatedIdentifierNameField: String = "input-delegated-0-0-name"
+  val delegatedIdentifierValueField: String = "input-delegated-0-0-value"
+  val delegatedAuthRuleField: String = "delegatedEnrolment[0].delegatedAuthRule"
+  val addDelegatedEnrolmentCTA: String = "[onclick='addDelegatedEnrolment()']"
 
```

Comment:
Formatting has changed the reading easiness. Can we use the formatting which is easy to read?